packages/odyssey-storybook/.storybook/preview-head.html

```diff
@@ -27,4 +27,24 @@
   .sbdocs .sbdocs-a {
     color: #1662dd;
   }
+
+  .sbdocs .sb-do::after,
+  .sbdocs .sb-dont::after {
+    display: inline-block;
+    margin-block-start: 1.14285714rem;
+    padding-block: 0.57142857rem;
+    padding-inline: 0.85714286rem;
+    border-radius: 1em;
+    color: #ffffff;
+  }
+
+  .sbdocs .sb-do::after {
+    content: "\2713 \2005 Do";
```

Comment:
minor, but this technique might render the content in a way that isn't accessible to some SR. Won't worry about it for now if it is flagged in a future audit we can prioritize it relative to actual component a11y work needed.